libkbfs/kbfs_ops.go

```diff
@@ -160,7 +160,12 @@ func (fs *KBFSOpsStandard) DeleteFavorite(ctx context.Context,
 		return fs.opsByFav[fav]
 	}()
 	if ops != nil {
-		return ops.deleteFromFavorites(ctx, fs.favs)
+		err := ops.deleteFromFavorites(ctx, fs.favs)
+		if _, ok := err.(OpsCantHandleFavorite); !ok {
```

Comment:
Could the `deleteFromFavorites` happen when `head == nil` because it's not initialized yet somehow but not because TLF doesn't exist? I was concerned if this would make it possible in any way to have favorites seemingly deleted on a device while it's not actually happened on server.